lib/galaxy/datatypes/triples.py

```diff
@@ -119,10 +115,7 @@ class Rdf( xml.GenericXml, Triples ):
     file_ext = "rdf"
 
     def sniff( self, filename ):
-        f = open( filename, "r" )
-        firstlines = "".join( f.readlines(5) )
```

Comment:
According to https://docs.python.org/2/library/stdtypes.html#file.readlines , `readlines()` only parameters is `sizehint`: "If the optional `sizehint` argument is present, instead of reading up to EOF, whole lines totalling approximately `sizehint` bytes (possibly after rounding up to an internal buffer size) are read."

So this was not the number of lines to read!

Similarly for `readline([size])`. I suppose we may just add `5 * 1024` to all `readline()/readlines()` calls and we are safe without creating new functions?